run_all_tests.bat

```diff
@@ -0,0 +1,45 @@
+@echo off
+
+rem Ask Java where its home
```

Comment:
I wonder if PowerShell will help with this script :-)

Reply:
I have no special affection for cmd.exe.
But this script is already ready, and I do not want to re-mark all the rakes :-)
But, if it gives more readable code, then it will make sense to rewrite it.

Reply:
After all I feel it much easier to rewrite it in Rust :-D